bitnami/thanos/values.yaml

```
@@ -1259,6 +1259,28 @@ storegateway:
   ##
   # existingConfigmap:
 
+  ## Thanos Store GRPC server configuration
+  ##
+  grpc:
+    ## TLS configuration
+    ##
+    tls:
+      # Enable TLS encryption in the GRPC server
+      enabled: false
+      # TLS Certificate for gRPC server, leave blank to disable TLS
+      cert:
+      # TLS Key for the gRPC server, leave blank to disable TLS
+      key:
+      # TLS CA to verify clients against. If no client CA is specified, there is no client verification on server side. (tls.NoClientCert)
+      ca:
+      # Existing secret containing your own TLS certificates.
+      # existingSecret:
+      #   name:
+      #   keyMapping:
+      #     ca-cert: ca.pem
+      #     tls-cert: cert.pem
+      #     tls-key: key.pem
```

Follow the same format used for comments in the rest of the file:
```suggestion
      ## Enable TLS encryption in the GRPC server
      ##
      enabled: false
      ## TLS Certificate for gRPC server, leave blank to disable TLS
      ##
      cert: ""
      ## TLS Key for the gRPC server, leave blank to disable TLS
      ##
      key: ""
      ## TLS CA to verify clients against. If no client CA is specified, there is no client verification on server side. (tls.NoClientCert)
      ##
      ca: ""
      ## Existing secret containing your own TLS certificates.
      ## Example:
      ## existingSecret:
      ##   name:
      ##   keyMapping:
      ##     ca-cert: ca.pem
      ##     tls-cert: cert.pem
      ##     tls-key: key.pem
      ##
      existingSecret: {}
```